src/components/interaction-designer/block-editors/BlockEditor.vue

```diff
@@ -49,6 +49,7 @@ export default {
 <style lang="scss">
 .block-editor-wrapper {
   position: absolute;
```

Comment:
the new bf `sticky` doesn't work in this context  😄

Reply:
😹